backend/.dockerignore

```diff
@@ -26,3 +26,4 @@ tests/
 # PCX
 pcr-backup
 frontend/
+tests/
```

Comment:
Tests are already in the dockerignore. Also technically everything in the Dev section and below can be removed without any issues since the images are only built in CircleCI